packages/syft/src/syft/service/worker/worker_image.py

```diff
@@ -18,64 +13,67 @@
 from ...types.syft_object import SYFT_OBJECT_VERSION_1
 from ...types.syft_object import SyftObject
 from ...types.uid import UID
-from ..response import SyftError
-from ..response import SyftSuccess
-
-
-def parse_output(log_iterator: Iterator) -> str:
```

Comment:
moved to `worker_image_service.py` function called `docker_push()`

Reply:
Can we move this function to `service/worker/utils.py` as well ?